src/CommonITILValidation.php

```diff
@@ -965,9 +964,9 @@ public static function processMassiveActionsForOneItemtype(
     /**
      * Print the validation list into item
      *
-     * @param CommonDBTM $item
+     * @param CommonITILObject $item
      **/
-    public function showSummary(CommonDBTM $item)
+    public function showSummary(CommonITILObject $item)
```

Comment:
If you change its signature, please make it private at the same time (it is used only in the current method) and add a line int he changelog for this change.

```suggestion
    private function showSummary(CommonITILObject $item)
```